src/catalog/catalog_accessor.cpp

```diff
@@ -79,6 +79,9 @@ bool CatalogAccessor::SetTablePointer(table_oid_t table, storage::SqlTable *tabl
 }
 
 common::ManagedPointer<storage::SqlTable> CatalogAccessor::GetTable(table_oid_t table) const {
+  if (UNLIKELY(catalog::IsTempOid(table))) {
+    return temp_tables_.find(table)->second;
```

Comment:
What happens if this lookup fails? If it can't we should store the result in an `UNUSED_ATTRIBUTE` local variable and then do an assert that it's not `end()`. All assumptions about code behavior should be asserted.